src/main/java/edu/ucsb/cs156/happiercows/controllers/UserCommonsController.java

```diff
@@ -99,26 +98,29 @@ public ResponseEntity<String> putUserCommonsByIdBuy(
 
         String body = mapper.writeValueAsString(userCommons);
         return ResponseEntity.ok().body(body);
-    }
+      }
 
-  @ApiOperation(value = "Sell a cow, totalWealth updated")
+  @ApiOperation(value = "Sell cows, totalWealth updated")
   @PreAuthorize("hasRole('ROLE_USER')")
   @PutMapping("/sell")
   public ResponseEntity<String> putUserCommonsByIdSell(
-          @ApiParam("commonsId") @RequestParam Long commonsId) throws NoCowsException, JsonProcessingException {
+    @ApiParam("commonsId") @RequestParam Long commonsId,
+    @ApiParam("numCows") @RequestParam int numCows) // Want to set 1 as the default when nothing is passed in
+      throws NoCowsException, JsonProcessingException {
         User u = getCurrentUser().getUser();
         Long userId = u.getId();
 
-        Commons commons = commonsRepository.findById(commonsId).orElseThrow( 
-          ()->new EntityNotFoundException(Commons.class, commonsId));
+        Commons commons = commonsRepository.findById(commonsId)
+          .orElseThrow(()->new EntityNotFoundException(Commons.class, commonsId));
         UserCommons userCommons = userCommonsRepository.findByCommonsIdAndUserId(commonsId, userId)
-        .orElseThrow(
-            () -> new EntityNotFoundException(UserCommons.class, "commonsId", commonsId, "userId", userId));
-
-
-        if(userCommons.getNumOfCows() >= 1 ){
-          userCommons.setTotalWealth(userCommons.getTotalWealth() + commons.getCowPrice());
-          userCommons.setNumOfCows(userCommons.getNumOfCows() - 1);
+          .orElseThrow(() -> new EntityNotFoundException(UserCommons.class, "commonsId", commonsId, "userId", userId));
+
+        if(userCommons.getNumOfCows() >= 1){
+          //if (numCows > userCommons.getNumOfCows()) numCows = userCommons.getNumOfCows();
+          //numCows = (numCows < userCommons.getNumOfCows()) ? numCows : userCommons.getNumOfCows();
```

Comment:
Why is there commented out code here?

We very much try to avoid merging commented out code into the main branch.

Reply:
I left it there because I felt the commented out code are both more clear than using the Math.min() function, but it causes pitest to fail when it changes > to >= since there is no way I can think of to write a test for that. I didn't know merging commented out code was a problem, so I will not do this in the future.

I am not sure how to fix it though, because this is not my branch.